extension-manifest-v2/app/content-scripts/renew.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { showIframe } from '@ghostery/ui/iframe';
+
+const HOUR_IN_MS = 1000 * 60 * 60;
+
+chrome.storage.local.get(['renew_setup'], ({ renew_setup }) => {
```

Comment:
this will fire for every page load. Would prefer to have it background triggered instead.

Reply:
Trigger the whole popup from the background might be tricky, as it would take an active tab, which might not be a HTTP(s) page - like a new tab page, settings, etc.. so then it won't be shown after the delay.

However, I agree that calling storage on each page load might not be the best idea. I will move it to the background with a cache, so the content script will trigger a message instead.